empress/reconcile/median.py

```diff
@@ -68,14 +68,14 @@ def mapping_node_sort(ordered_gene_node_list, ordered_species_node_list, mapping
     return sorted_list
 
 
-def generate_scores(preorder_mapping_node_list, dtl_recon_graph, gene_root, normalize=True):
+def generate_frequencies_dict(preorder_mapping_node_list, dtl_recon_graph, gene_root, normalize=True):
     """
     Computes frequencies for every event
     :param preorder_mapping_node_list: A list of all mapping nodes in DTLReconGraph in double preorder
     :param dtl_recon_graph: The DTL reconciliation graph that we are scoring
     :param gene_root: The root of the gene tree
     :return: 0. A file structured like the DTLReconGraph, but with the lists of events replaced
```

Comment:
I think we are not actually explicitly constructing a "scored graph" with mapping nodes as keys and events and their frequencies as values, so this comment can be misleading